sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -2,7 +2,6 @@
   <!-- This component is shared between ministry and student users -->
   <body-header
```

Comment:
Is the `<body-header-container>` not applicable here also?

Reply:
I added it in the `Locations` component, although not sure if that is the best way of doing it.

```
<template>
  <tab-container>
    <body-header-container>
      <location-summary
        :institutionId="institutionId"
        @editLocation="gotToEditLocation"
      />
    </body-header-container>
  </tab-container>
</template>
```